src/features/channels/components/chat-content/message/Message.styles.ts

```diff
@@ -73,6 +77,24 @@ const useStyles = makeStyles((theme: Theme) =>
         textDecoration: 'underline',
       },
     },
+    // speechBubble: {
```

Comment:
Please do not commit commented code. If we still need to commit commented code for a further task, we should add a TODO comment with the task id so we can keep track of it.